samples/plugins/mysql_odbc/connection-dialog.tcd

```diff
@@ -0,0 +1,10 @@
+<connection-dialog class='mysql_odbc'>
```

Comment:
All of the other files class value should match the Manifest class value.  Packaging tool will check this.